### PR TITLE
Updated to use browserslist.

### PR DESCRIPTION
**Pull request by eclectic-coding**
Updated Gulp Workflow to use browserslist.

### CODE REVIEW

**Comment by cferdinandi** [comment]
I really appreciate this PR, but it looks like you also reformatted the files in the process, changing tabs to spaces, removing semicolons, and so on. 

 If you want to resubmit with only the things that actually changed modified, I will happily merge.

**Comment by eclectic-coding** [comment]
I will look at this afternoon and format with prettier. Sometimes this happens between editors.

**Comment by eclectic-coding** [comment]
I think this fixes the formatting issues. If not let me know.

**Comment by cferdinandi** [comment]
I'm not seeing any changes. The tabs are now spaces, the semicolons are gone, etc. You can click on the "Files Changed" tab above to see the differences.

**Comment by eclectic-coding** [comment]
Hopefully this fixes the errors in the formatting. The editor formatting issues are frustrating.

**Comment by cferdinandi** [comment]
Still seeing them in package.json. Does your editor auto-reformat or something?

File: package.json [hunk 1]
```diff
@@ -35,6 +35,12 @@
 		"gulp-cssnano": "2.1.3",
 		"gulp-autoprefixer": "6.0.0",
 		"gulp-svgmin": "2.1.0",
-		"browser-sync": "2.26.3"
-	}
+		"browser-sync": "2.26.3",
+		"browserlist": "^1.0.0",
```
Comment (cferdinandi) [inline package.json]: This should be a fixed version

Comment (eclectic-coding) [reply package.json]: Yeh, I just noticed that.

**Comment by cferdinandi** [comment]
Because of the errors on this PR, I manually updated with another PR

**Pull request closed without merging**